File: app/src/main/java/com/mazanca/newrespiracao/model/MiniPauser.java
package com.mazanca.newrespiracao.model;

public class MiniPauser implements Respirar{
    @Override
    public String getNome() {
        return "Mini Pauser";
    }

    @Override
    public long getTempoInspirar() {
        return 2;
    }

    @Override
    public long getTempoExpirar() {
        return 3;
    }

    @Override
    public long getNumeroDeCiclos() {
        return 1;
    }

    @Override
    public long getTempoPausa() {
        return 1;
    }
}

File: app/src/main/java/com/mazanca/newrespiracao/model/MiniPauser.kt
package com.mazanca.newrespiracao.model;

import kotlinx.parcelize.Parcelize
import android.os.Parcelable

@Parcelize
data class MiniPauser(private val tempoPausaCustom: Long = 4) : Respirar, Parcelable {
    override fun getNome() = "Mini Pausa"
    override fun getTempoInspirar() = 2L
    override fun getTempoExpirar() = 4L
    override fun getNumeroDeCiclos() = 3L
    override fun getTempoPausa() = tempoPausaCustom
}



